observability/logging/logging-6.0/log6x-opentelemetry-data-model.adoc

```diff
@@ -0,0 +1,375 @@
+
+You can forward these streams according to the sementatic conventions defined by OpenTelemetry semantic attributes. The semantic conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry, identified by *Attributes*. For example, a process running in a container includes attributes such as container_name, cluster_id, pod_name, namespace, and possibly deployment or app_name. These *Attributes* are grouped in the *Resource* object, which helps reduce repetition and optimizes log transmission as telemetry data.
+
+In addition to *Resource* attributes, logs might also contain *Scope* attributes specific to instrumentation libraries and *Log* attributes specific to each log entry. These attributes provide greater detail about each log entry and enhance filtering capabilities when querying logs in storage.
+
+The following sections define the attributes that are generally forwarded.
+
+=== Log Entry Structure
+
+All log streams include the following link:https://opentelemetry.io/docs/specs/otel/logs/data-model/#log-and-event-record-definition[log data] fields:
+
+The *Applicable Sources* column indicates which log sources each field applies to:
+
+* `all`: This field is present in all logs.
+* `container`: This field is present in Kubernetes container logs, both application and infrastructure.
+* `audit`: This field is present in Kubernetes, OpenShift API, and OVN logs.
+* `auditd`: This field is present in node auditd logs.
+* `journal`: This field is present in node journal logs.
+
+[cols="1,1,1", options="header"]
+|===
+|Name |Applicable Sources |Comment
+
+|`body`
+|all
+|
+
+|`observedTimeUnixNano`
+|all
+|
+
+|`timeUnixNano`
+|all
+|
+
+|`severityText`
+|container, journal
+|
+
+|`attributes`
+|all
+|(Optional) Present when forwarding stream specific attributes
+|===
+
+=== Attributes
+
+Log entries include a set of resource, scope, and log attributes based on their source, as described in the following table.
+
+The *Location* column specifies the type of attribute:
+
+* `resource`: Indicates a resource attribute
+* `scope`: Indicates a scope attribute
+* `log`: Indicates a log attribute
+
+The *Storage* column indicates whether the attribute is stored in a LokiStack using the default `openshift-logging` mode and specifies where the attribute is stored:
+
+* `stream label`: 
+** Enables efficient filtering and querying based on specific labels.
+** Can be labeled as *required* if the Loki Operator enforces this attribute in the configuration.
+* `structured metadata`: 
+** Allows for detailed filtering and storage of key-value pairs.
+** Enables users to use direct labels for streamlined queries without requiring JSON parsing.
+
+With OTLP, users can filter queries directly by labels rather than using JSON parsing, improving the speed and efficiency of queries.
+
+[cols="1,1,1,1,1", options="header"]
+|===
+|Name |Location |Applicable Sources |Storage (LokiStack) |Comment
```

Comment:
The data-model repository has changed. The changes from rhobs/observability-data-model#7 already seem to be included in this PR, but yesterday's changes from rhobs/observability-data-model#8 ("level") are still needed.